exercises/run-length-encoding/src/example.c

```diff
@@ -0,0 +1,130 @@
+#include "run_length_encoding.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <ctype.h>
+#include <math.h>
+
+enum rw_mode { READ, WRITE };
+
+/* count_digits: return the count of digits in a number */
+static size_t count_digits(size_t num)
+{
+   return log10(num) + 1;
+}
+
+/**
+ * enc_helper: calculate the encoded size of 'text';
+ *             encode in 'encoded' if write mode is set
+ */
+static size_t enc_helper(char *encoded, const char *text,
+                         const enum rw_mode mode)
+{
+   size_t enc_len = 0;
+   while (*text) {
+      if (isdigit(*text)) {
+         fprintf(stderr, "Digits in the plaintext are not supported!\n");
+         return 0;
+      }
```

Comment:
Is there a test for this assertion?

Reply:
Nope. I added this for the sake of completeness. 

Following [your comment in the other PR](https://github.com/exercism/c/pull/281#pullrequestreview-101385711), will remove this too.